index.html

```diff
@@ -4264,6 +4264,48 @@ <h3>Spectrum of Privacy</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h3>Software Trust Boundaries</h3>
+
+        <p>
+There are a variety of trust boundaries that exist in the
+<a href="#ecosystem-overview">ecosystem described by this specification</a>.
+Just like an individual using a web browser trusts the web browser not to betray
+their trust by uploading their personal information to a data broker, a similar
+trust exists between the roles in the ecosystem described by this specification
+and the software that operates on the behalf of each of those roles. For
+example,
+        </p>
+
+        <ul>
+          <li>
+An <a>issuer</a> trusts the issuer software it uses to not issue <a>verifiable
+credentials</a> without its permission.
+          </li>
+          <li>
+A <a>verifier</a> trusts the verifier software it uses to be truthful regarding
+the verification status of the <a>verifiable credentials</a> and
+<a>verifiable presentations</a> that it is checking.
+          </li>
+          <li>
+A <a>holder</a> trusts the digital wallet software it uses to not divulge
+information to a <a>verifier</a> without its consent.
+          </li>
+        </ul>
+
+        <p>
+The examples above are not exhaustive, and there can be a variety of other
+expectations that roles have of the software that they use to achieve their
+goals. The common thread between each of these expectations is that each
+role expects the software that it uses to not violate privacy expectations.
+That is, it is expected that the software will operate in the best interests
+of the role and a violation of those expectations is a violation of trust that
```

Comment:
It would be useful to mention how conflicts of interest may be represented. In many cases, there's likely to be software that represents many different parties and it should be recognized that a software cannot act as a UA for multiple parties at the same time.

Reply:
Agreed - I'm not saying that we should forbid it but rather to recognize that the conflict of interests often times can lead to the collapse of the trust boundaries. This is part of the value in having these external reviews by judges in the legal system where as for us we don't really have any such authority to require that within the technical specification. Hence not really pushing for this to be normatively required, but rather to call out that a UA isn't necessarily always a UA if it's answering to two competing parties. Eventually one will have to take priority over the other.

Reply:
I made an attempt at some text in 7fce91ea1a3a70feab9b60c00daf2b4dcec41ff5.

Reply:
++ that addresses my concerns